controls/stig_rhel9.yml

```diff
@@ -958,6 +958,7 @@ controls:
             - medium
         title: RHEL 9 cron configuration directories must have a mode of 0700 or less permissive.
         rules:
+            - package_crontabs_installed
```

Comment:
This isn't in the STIG. Why should be added?

See https://stigaview.com/products/rhel9/v2r2/RHEL-09-232040/

Reply:
Good point! I have changed the PR to add `package_cron_installed` instead.